node/cli/src/command.rs

```diff
@@ -47,6 +47,7 @@ impl IdentifyChain for dyn sc_service::ChainSpec {
 
 impl PartialEq for ChainIdentity {
 	fn eq(&self, other: &Self) -> bool {
+		#[allow(clippy::match_like_matches_macro)]
```

Comment:
I almost disabled this lint entirely, but the new `matches!` macro is nice. Just not in a few more complex places.